gitlab/misc.sh

```diff
@@ -177,7 +177,7 @@ install_conftest() {
 
   echo "Installing ConfTest ${CONFTEST_VERSION}"
 
-  wget -q https://github.com/instrumenta/conftest/releases/download/v${CONFTEST_VERSION}/conftest_${CONFTEST_VERSION}_Linux_x86_64.tar.gz -O /tmp/conftest.tar.gz
+  wget -q https://github.com/open-policy-agent/conftest/releases/download/v${CONFTEST_VERSION}/conftest_${CONFTEST_VERSION}_Linux_x86_64.tar.gz -O /tmp/conftest.tar.gz
```

Comment:
I tried this locally against `v0.19.0` and it failed ..hmm

Reply:
works for me :P